src/metabase/api/common/openapi.clj

```diff
@@ -57,7 +61,8 @@
                                               (= (second (:oneOf v)) {:type "null"}))
                                          (do
                                            (swap! not-required conj k)
-                                           (assoc (first (:oneOf v)) :description (:description v)))
+                                           (merge (first (:oneOf v))
+                                                  (select-keys v [:description :default])))
 
                                          (= (:enum v) ["true" "false" true false])
```

Comment:
I know you didn't modify this, but the true false true false vector here feels like a magic value. Are you open to tying it to an actual value somewhere? I just have a spider sense this will completely break once we add "True" and "fAlSe" to it.

Reply:
You're right, but tests will fail (they check exactly for that) 😁 I'll reference BooleanValue though.